src/services/AppService.ts

```diff
@@ -13,16 +15,26 @@ interface AppServiceInterface {
     gatewayUID: string,
     sensorUID: string
   ) => Promise<SensorReading<unknown>[]>;
+  setSensorName: (
```

Comment:
I know this isn't new, but I'm not totally aligned on using a app facade here, seems like a rod for our own back. Fortunately it's not costing us much to maintain it.

Reply:
The thought was to have a pure JS object that does everything our app needs to do from a user's perspective - imagine we wanted to make a CLI version of the website. Following Bob Martin's Clean Architecture, this would be called an Interactor or and should represent the high level business rules as use cases. Technically, he would also have you put critical business rules into an even-higher level set of _Entities_ which have nothing to do with use cases. But I haven't seen need for Entities yet. Basically trying to make sure we have _something_ in the red circle here:
![image](https://user-images.githubusercontent.com/22032748/155262986-cf91bb8f-31a6-4257-89ec-6db6d75fdbbf.png)

Reply:
Without the AppService, what would you consider the highest-level component(s) in the system?

I imagine the AppService (use cases) will get more complicated as we add things like auth or notifications. When we need to integrate more than just notehub stuff for the benefit of our uses.

Reply:
No need for change right now, just felt like an itch to scratch ;-). The explanations you've given to my comments here and above would be really useful in a design doc so they don't get lost in a closed PR.